HOWTO-RELEASE

```diff
@@ -32,47 +45,191 @@
    - If any interfaces have been removed since the last public release, then
      set age to 0.
 
-4.5) Run "autogen.sh" (hopefully on the same machine it was last run on)
+Update the following files with the new ABI version number:
+
+    - `src/Makefile.am`     Update -version-info
+    - `CMakelists.txt`:     Update PROJ_BUILD_VERSION (cur:rev:age) and
+                            PROJ_API_VERSION (current)
+
+*Commit the changes to master.*
+
+
+1.2 Update man pages
+-------------------------------------------------------------------------------
+
+Man pages are (mostly) automatically generated from the Sphinx docs. The pages
+in man/man3/ was manually updated in 1.1. Here we update the pages in
+man/man1/. Follow these steps:
+
+    - Update version number and date in `man/man3/pj_init.3` and
+      `man/man3/geodesic.3`
+    - Temporarily update version in `docs/source/conf.py`
+    - run `script/update_man.sh`
+    - Revert version number in `docs/source/conf.py`
+
+The `update_man.sh` script builds the man pages and commits the update man pages
+to master.
+
+
+1.3 Write release notes
+-------------------------------------------------------------------------------
+
+Update `NEWS` with descriptions of the changes since the last release.
+Hopefully issues and pull requests that go into the new release have been
+properly tagged with the milestone for the current release.
+
+End the section by thanking the authors that contributed to the release. Get a
+sorted list of contributors in descending order of activity:
+
+```
+git shortlog -sn x.y.z..HEAD | tac | awk '{print $2 " " $3}'
+```
+
+where `x.y.z` is the tag of the previous release. You may need to edit the list
+slightly since not all authors has properly configured their names in git.
+
+*Commit the changes to master.*
+
+
+1.4 Update `CITATION`
+-------------------------------------------------------------------------------
+
+If needed, update the year in `CITATION`.
```

Comment:
the citation is also repeated in `docs/source/about.rst`, so perhaps add that here too.